app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -13,6 +13,8 @@ img, iframe
 .list-answer
   display: none
 
+.project-search-image
+  max-width: 50px !important
```

Comment:
@diogob I couldn't figure out why it was there in the first place, everything seems fine with redactor without using !important. I have removed it for now, we can review this later if any problems arise.